.github/actions/node_env_setup/action.yml

```
@@ -1,4 +1,4 @@
-name: Set up testing infrastructure 
+name: Set up NodeJS environment 
```

This is the brand spelling. Please correct throughout.

```suggestion
name: Set up Node.js environment
```